op-e2e/actions/proofs/isthmus_fork_test.go

```diff
@@ -427,4 +441,19 @@ func TestIsthmusNetworkUpgradeTransactions(gt *testing.T) {
 	latestBlock, err = ethCl.BlockByNumber(context.Background(), nil)
 	require.NoError(t, err)
 	checkRecentBlockHash(latestBlock.NumberU64()-1, latestBlock.Header().ParentHash, "post-activation")
+
+	l2Safe := sequencer.L2Safe()
```

Comment:
same, need to check correct block

Reply:
[3f0bc0d](https://github.com/ethereum-optimism/optimism/pull/15296/commits/3f0bc0da3a324a8c01674e7f6a8e303579055c1b)